src/lib/get-property-id.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import { getMerchantID, getSDKQueryParam } from '@paypal/sdk-client/src';
+import { getDisableSetCookie, getMerchantID, getSDKQueryParam } from '@paypal/sdk-client/src';
```

Comment:
`getDisableSetCookie` is confusing ... maybe `isDisableCookie` or `isCookieDisabled` ? Is this a function we own and can change ?

Reply:
We don't own it. It is part of js SDK.
I believe they choose this name because in some point they are passing header `disable-set-cookie` or query parameter `disableSetCookie=true`. For this header name this function name make sense. 

I don't know why they choose this name, might be inside `stargate` module witch set extra cookies have name `set-cookie`